.github/workflows/lint.yaml

```diff
@@ -23,8 +22,4 @@ jobs:
       - name: Format
         run: cargo fmt --all -- --check
 
-      - name: Clippy
-        uses: actions-rs/clippy-check@b5b5f21f4797c02da247df37026fcd0a5024aa4d # v1.0.7
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-targets
```

Comment:
This pull request follows the original action's behavior of not failing when there's a linting issue.